Sources/Rosetta/CardSets/Expert1CardsGen.cpp

```diff
@@ -3,10 +3,19 @@
 // RosettaStone is hearthstone simulator using C++ with reinforcement learning.
 // Copyright (c) 2019 Chris Ohk, Youngjoong Kim, SeungHyun Jeon
 
+#include <Rosetta/Cards/Cards.hpp>
+#include <Rosetta/CardSets/CoreCardsGen.hpp>
```

Comment:
Maybe it seems to be unnecessary header.

Reply:
Fixed.